src/answers-search-bar.js

```diff
@@ -252,12 +253,15 @@ class AnswersSearchBar {
     }
     parsedConfig.sessionTrackingEnabled = sessionTrackingEnabled;
 
-    const authIdKey = parsedConfig.apiKey ? 'apiKey' : 'token';
-    const sandboxPrefix = `${SANDBOX}-`;
-    parsedConfig[authIdKey].includes(sandboxPrefix)
-      ? parsedConfig.environment = SANDBOX
-      : parsedConfig.environment = PRODUCTION;
-    parsedConfig[authIdKey] = parsedConfig[authIdKey].replace(sandboxPrefix, '');
+    if (parsedConfig.apiKey) {
```

Comment:
so basically we don't need to check for the sandbox- prefix if a token is used instead of an apiKey?

Reply:
yeah, I believe token will never have a prefix. We only need to take care of apiKey prefix for the purpose of backward compatibility